signac/__main__.py

```diff
@@ -637,7 +635,7 @@ def main_update_cache(args):
 # UNCOMMENT THE FOLLOWING BLOCK WHEN THE FIRST MIGRATION IS INTRODUCED.
```

Comment:
I just realized this PR is targeting `next`. The `next` branch is no longer up to date, and this PR should be targeting `master`.